lua/octo/init.lua

```diff
@@ -621,6 +652,8 @@ function M.create_issue_buffer(issue, repo, create_buffer)
     200
   )
 
+  async_fetch_taggable_users(bufnr, repo, issue.user.login)
```

Comment:
It might be cleaner to just not pass `issue.user.login` into this function since I am already setting that value as the default for `taggable_users` [here](https://github.com/pwntester/octo.nvim/pull/48/files#diff-551b860db1f227d6d4d20dfc8a8a09669a68b474e514e85baf9f22c2b4a85e04R474). Then inside the function I can get that variable and append to it. Thoughts?

Reply:
I think thats a better approach, better get `taggable_users` from the buf vars and add new users to it.

Reply:
This should be resolved [here](https://github.com/pwntester/octo.nvim/pull/48/commits/f6771a23b5f6126baaba6a7e7312daaa14127e7b)